From 9ce87ad70435b397450af43d7d5672a6c958159c Mon Sep 17 00:00:00 2001
From: Radu Bulie <radu.bulie@freescale.com>
Date: Tue, 5 Nov 2013 13:18:03 +0000
Subject: [PATCH 325/429] dpa_offload: Add correct macless tag in 2041 dts
 file

Replace "fsl,dpa-ethernet" with "fsl,dpa-ethernet-macless"
in order to initialize the macless interfaces.

Signed-off-by: Radu Bulie <radu.bulie@freescale.com>
Change-Id: Idc9ad966df6727d411c03b0d29414dfca3271067
Reviewed-on: http://git.am.freescale.net:8181/6278
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Marian-Cornel Chereji <marian.chereji@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
[origin patch is from QorIQ-SDK-V1.5-20131219-yocto]
Signed-off-by: Yang Wei <Wei.Yang@windriver.com>
---
 .../dts/p2041rdb-usdpaa-shared-interfaces.dts      |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dts/p2041rdb-usdpaa-shared-interfaces.dts b/drivers/staging/fsl_dpa_offload/dts/p2041rdb-usdpaa-shared-interfaces.dts
index b751aa4..106218e 100644
--- a/drivers/staging/fsl_dpa_offload/dts/p2041rdb-usdpaa-shared-interfaces.dts
+++ b/drivers/staging/fsl_dpa_offload/dts/p2041rdb-usdpaa-shared-interfaces.dts
@@ -119,14 +119,14 @@
 		 * On receiving frame at this interface, the ethernet driver will do
 		 * kmap_atomic/kunmap_atomic for that frame. */
 		ethernet@16 {
-			compatible = "fsl,p2041-dpa-ethernet", "fsl,dpa-ethernet";
+			compatible = "fsl,p2041-dpa-ethernet", "fsl,dpa-ethernet-macless";
 			fsl,bman-buffer-pools = <&bp16>;
 			fsl,qman-frame-queues-rx = <4000 8>;
 			fsl,qman-frame-queues-tx = <4008 8>;
 			local-mac-address = [00 11 22 33 44 55];
 		};
                 ethernet@17 {
-                        compatible = "fsl,b4860-dpa-ethernet", "fsl,dpa-ethernet";
+                        compatible = "fsl,p2041-dpa-ethernet", "fsl,dpa-ethernet-macless";
                         fsl,bman-buffer-pools = <&bp16>;
                         fsl,qman-frame-queues-rx = <5000 8>;
                         fsl,qman-frame-queues-tx = <5008 8>;
-- 
1.7.5.4

